src/frontend-pwa/src/content/content.ts

```diff
@@ -68,6 +68,42 @@ export const SettingsContent: ContentMap = {
     eng: 'Pull in new location data from the server',
     fr: 'Récupérer de nouvelles données de localisation à partir du serveur',
   },
+  refreshDataButtonTextRefresh: {
+    eng: 'Refresh',
+    fr: 'Rafraîchir',
+  },
```

Comment:
I think this would be better off with something like "Install Maps", refresh doesn't give much insight to what that means in relation to Offline maps